README.md

```diff
@@ -238,41 +298,78 @@ config :bamboo, :json_library, SomeOtherLib
 
 ## Testing
 
-You can use the Bamboo.TestAdapter along with [Bamboo.Test] to make testing your
-emails straightforward.
+Bamboo separates email creation and email sending. Test email creation by
+asserting against the email struct created by your functions. For example,
+assuming your welcome email accepts a user recipient, provides the correct from
+address, and provides specific text, you might test like this:
 
 ```elixir
-defmodule MyApp.RegistrationTest do
+defmodule MyApp.EmailTest do
   use ExUnit.Case
-  use Bamboo.Test
-  alias MyApp.Email
 
-  # Unit testing is easy since the email is just a struct
   test "welcome email" do
-    user = new_user
+    user = {"Ralph", "ralph@example.com"}
 
-    email = Email.welcome_email(user)
+    email = MyApp.Email.welcome_email(user)
 
     assert email.to == user
-    # The =~ checks that the html_body contains the text on the right
-    assert email.html_body =~ "Thanks for joining"
+    assert email.from == "welcome@myapp.com"
+    assert email.html_body =~ "<p>Thanks for joining</p>"
+    assert email.text_body =~ "Thanks for joining"
   end
+end
+```
+
+Test email sending in integration tests by using the [`Bamboo.TestAdapter`]
+along with [`Bamboo.Test`]. For example, assuming during the registration
+process of your app an email is sent to the user welcoming them to the
+application, you might test this feature like this:
+
+```elixir
+defmodule MyApp.RegistrationTest do
+  use ExUnit.Case
+  use Bamboo.Test
+
+  # Remember to use the `Bamboo.TestAdapter` in your test config
 
-  # Integration test with the helpers from Bamboo.Test
   test "after registering, the user gets a welcome email" do
-    user = new_user
+    user = new_user()
+    expected_email = MyApp.Email.welcome_email(user.email)
 
     MyApp.Registration.create(user)
 
-    assert_delivered_email Email.welcome_email(user)
+    assert_delivered_email expected_email
+  end
+
+  defp new_user do
+    # Build a user appropriate to your application
   end
 end
 ```
 
-See documentation for [Bamboo.Test] for more examples, and remember to use
-Bamboo.TestAdapter.
+See the documentation for [`Bamboo.Test`] for more examples and additional
+helper functions.
+
+## Available Adapters
 
-[bamboo.test]: https://hexdocs.pm/bamboo/Bamboo.Test.html
+The [`Bamboo.MandrillAdapter`] and [`Bamboo.SendGridAdapter`] **are being used
```

Comment:
I feel like calling out that these are actually being used by real life people sorta makes it seem like we are surprised by that fact 😅 Maybe we can just omit this opening paragraph?

Reply:
I, too, was hesitant about this. I'm guessing it was included initially to instill confidence in those looking for a library. I think at this point Bamboo is well established enough that removing it is the better option.